ui/widgets/autocomplete.js

```diff
@@ -614,6 +614,22 @@ $.widget( "ui.autocomplete", {
 			// Prevents moving cursor to beginning/end of the text field in some browsers
 			event.preventDefault();
 		}
+	},
+
+	// Support: Chrome <=50
+	// We should be able to just use this.element.prop( "isContentEditable" )
+	// but hidden elements always report false in Chrome.
+	// https://code.google.com/p/chromium/issues/detail?id=313082
+	_isContentEditable: function( element ) {
+		if ( !element.length ) {
+			return false;
+		}
+
+		var editable = element.prop( "contentEditable" );
+
+		return editable === "true" ? true :
+			editable === "false" ? false :
+			this._isContentEditable( element.parent() );
```

Comment:
This seems fine to me:
```js
if ( editable === "inherit" ) {
  return this._isContentEditable( element.parent() );
}

return editable === "true";
```
Assuming "true", "false" and "inherit" are the only values `editable` ever has, that should behave exactly like yours.